screens/EditSignal/index.tsx

```diff
@@ -64,6 +73,8 @@ export default function EditSignal() {
         title="Save"
```

Comment:
for when the button is loading we can have something like 
```suggestion
        title={loading?"Saving...":"Save"}
```